### PR TITLE
Add test for unaddressed

### PR DESCRIPTION
**Pull request by NickGrantX**
# Motivation and Context
We need to print a certain amount of letters, questionnaires, continuation pages etc. without an address. These will have an associated UAC/QID but no case - the case association will happen later. This PR covers the part where we generate the UAC/QID pair and emit the event to RH and Action Scheduler.

# What has changed
Added test scenario for unaddressed.

# How to test?
Run the acceptance tests against the Case Processor branch `rmc-176-uac-qid-pairs-for-unaddressed`.

# Links
Trello: https://trello.com/c/Hv7z7OqL/684-rmc-176-create-uac-qid-pairs-for-unaddressed-ccs-questionnaires-8

# Screenshots (if appropriate):

### CODE REVIEW

**Comment by NickGrantX** [comment]
[RMC-176 Create UAC/QID pairs for unaddressed/CCS questionnaires (8)](https://trello.com/c/Hv7z7OqL/684-rmc-176-create-uac-qid-pairs-for-unaddressed-ccs-questionnaires-8)

File: acceptance_tests/features/steps/unaddressed_steps.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+from acceptance_tests.utilities.rabbit_helper import start_listening_to_rabbit_queue
+from acceptance_tests.utilities.sample_loader.rabbit_context import RabbitContext
+from config import Config
+
+tc = TestCase('__init__')
+
+
+@given('the unaddressed batch is run')
+def unaddressed_batch_run(context):
+    return
```
Comment (AdamHawtin) [inline acceptance_tests/features/steps/unaddressed_steps.py]: Could just delete this step from the scenario?

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
Passing locally and in k8s against the case processor branch for me, I just think the empty step should be deleted.

File: acceptance_tests/features/steps/unaddressed_steps.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import json
+from unittest import TestCase
+
+from behave import given, when, then
+
+from acceptance_tests.utilities.rabbit_helper import start_listening_to_rabbit_queue
+from acceptance_tests.utilities.sample_loader.rabbit_context import RabbitContext
+from config import Config
+
+tc = TestCase('__init__')
```
Comment (AdamHawtin) [inline acceptance_tests/features/steps/unaddressed_steps.py]: We do this in a couple of places now, worth factoring it out into a helper?

File: acceptance_tests/features/steps/unaddressed_steps.py (hunk 1 above)
Comment (NickGrantX) [reply acceptance_tests/features/steps/unaddressed_steps.py]: Is it even legal to have a "Given... Then" without a "When"?

Comment (AdamHawtin) [reply acceptance_tests/features/steps/unaddressed_steps.py]: It makes sense as a "When... Then". We've done that in a few other places where there isn't any data/state setup required in the business/social tests, behave doesn't enforce the "Given, When, Then" pattern.

File: acceptance_tests/features/steps/unaddressed_steps.py (hunk 2 above)
Comment (NickGrantX) [reply acceptance_tests/features/steps/unaddressed_steps.py]: it's a one-liner... as would the helper be?

**Review by rjweeks — APPROVED** [review]

File: acceptance_tests/features/steps/unaddressed_steps.py (hunk 1 above)
Comment (NickGrantX) [reply acceptance_tests/features/steps/unaddressed_steps.py]: ok will change then

File: acceptance_tests/features/steps/unaddressed_steps.py (hunk 2 above)
Comment (NickGrantX) [reply acceptance_tests/features/steps/unaddressed_steps.py]: well if Monsier Jamie agrees I'm outnumbered - will refactor

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**